Formula/s-nail.rb

```diff
@@ -11,14 +11,12 @@ class SNail < Formula
     sha256 "4d137ea6f6ff75f6bd792584aa4d86ead5445f0e5df9c9ed7074d84df3b7fb0f" => :el_capitan
   end
 
+  depends_on "awk"
```

Comment:
Can we make it a build-only dependency?

```suggestion
  depends_on "awk" => :build
```

Reply:
Changed as requested.